### ISSUE TITLE
Archive.today - a time capsule for web pages!

### ISSUE THREAD

**Issue by UjwalKandi** [UjwalKandi/Issue-Bookmarks#514]
**TL;DR** - It takes a 'snapshot' of a webpage that will always be online even if the original page disappears. It saves a text and a graphical copy of the page for better accuracy and provides a short and reliable link to an unalterable record of any web page.

This can be useful if you want to take a 'snapshot' of a page which could change soon: price list, job offer, real estate listing, drunk blog post. Saved pages will have no active elements and no scripts, so they keep you safe as they cannot have any popups or malware!

**Website:**
https://archive.is/
[https://archive.is/2020.04.21/rt.live/](https://archive.is/2020.04.21/https://rt.live/)
[https://archive.is/2014.06.26/google.com/maps/…](https://archive.is/2014.06.26/https://www.google.com/maps/@-12.1858563,96.8293918,366m/data=!3m1!1e3)
